dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ranger/RangerConnector.java

```diff
@@ -127,14 +123,14 @@ protected String toCallDescription() {
     }
   }
 
-  static class DumpGroupsTask extends AbstractRangerTask<Group> {
+  static class DumpGroupsTask extends AbstractRangerTask {
 
     DumpGroupsTask() {
       super(GroupsFormat.ZIP_ENTRY_NAME);
     }
 
     @Override
-    protected Iterator<Group> dataIterator(@Nonnull RangerClientHandle handle) {
+    protected Iterator<Object> dataIterator(@Nonnull RangerClientHandle handle) {
       return new RangerPageIterator<>(
           page -> {
             try {
```

Comment:
Should the try/catch be in RangerPagerIterator.computeNext()? It would have to use its own FunctionalInterface, but that's no cost. It would make all these callers much cleaner, and reduce the number (if not the call-frequency) of 'throw new RuntimeException' calls, which is generally an antipattern.

Might be out of the scope of this PR though, as you didn't change that.

Reply:
Good idea - done.